packages/common/core-interfaces/src/handles.ts

```diff
@@ -100,3 +100,26 @@ export interface IFluidHandle<
 	 */
 	bind(handle: IFluidHandle): void;
 }
+
+/**
+ * Check if a value is an IFluidHandle.
+ * @remarks
+ * Objects which have a field named `IFluidHandle` can in some cases produce a false positive.
+ * @internal
+ */
+export function isFluidHandle(value: unknown): value is IFluidHandle {
```

Comment:
This package does not have tests.
Should I:

1. Move this to a different package which does have tests (if so, which package?).
2. Have tests in another package's tests. (If so which package?)
3. Add tests to this package.

Reply:
core-utils is for "This package is intended for sharing and promoting best-practice implementations of Fluid-agnostic utility functions across packages in the Fluid Framework repo." (source: core-utils readme). This function is not Fluid agnostic, and putting it in core-utils would make it depend on core-interfaces, which I don't think makes sense given core-utils spec.

Reply:
Interesting. I did not know that that. I don't have any good suggestions, sorry.
The `@fluidframework/runtime-utils` comes to mind which has things that are used at runtime and below layers.

Reply:
From what I can find, maybe @fluidframework/shared-object-base is the best option since all the usages already depend on that (one is in there)?

@fluid-internal/client-utils seems out since we might want users outside of fluid to have it eventually.

"@fluidframework/container-runtime" seems like an option:  It has no top level comment in its readme saying what goes it, and it already uses handles.

Reply:
runtime utils has packages/runtime/runtime-utils/src/handles.ts has isSerializedHandle which seems like a good place for this to live as well. Moving....

Reply:
Done.